src/Cli/dotnet/commands/dotnet-workload/install/FileBasedInstaller.cs

```diff
@@ -702,5 +702,12 @@ private bool PackHasInstallRecords(PackInfo packInfo)
         }
 
         private bool IsSingleFilePack(PackInfo packInfo) => packInfo.Kind.Equals(WorkloadPackKind.Library) || packInfo.Kind.Equals(WorkloadPackKind.Template);
+
+        public bool GetWorkloadInstallMode(SdkFeatureBand sdkFeatureBand)
+        {
+            string path = Path.Combine(WorkloadInstallType.GetInstallStateFolder(sdkFeatureBand, _dotnetDir), "default.json");
```

Comment:
Probably don't need to abstract through IInstaller because the WorkloadInstallType.GetInstallStateFolder call is the same for both types (as is the rest of the code)